dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.restfulWS3.1-mpMetrics-monitor1.0.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.restfulWS3.1-mpMetrics-monitor1.0
+Manifest-Version: 1.0
+IBM-Provision-Capability: \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.monitor-1.0))", \
```

Comment:
```suggestion
```

Since monitor is already enabled by mpMetrics-5.0 no need to have it in the auto feature's list of features.